libompd/src/omp-icv.cpp

```diff
@@ -388,6 +391,8 @@ static ompd_rc_t ompd_get_nthreads(
   }
 
   return ompd_rc_ok;
+}
```

Comment:
Why add a close-brace here? I don't see a matching open-brace above. Is this fixing a previous commit?

Reply:
Yes, this was an accidental merge mistake. Correcting it here.